vendor/github.com/BurntSushi/toml/session.vim

```diff
@@ -0,0 +1 @@
+au BufWritePost *.go silent!make tags > /dev/null 2>&1
```

Comment:
Is this part of the toml package? Seems like an accidentally commit.

Reply:
I certainly didn't write it, seems like maybe they committed it accidentally.

https://github.com/BurntSushi/toml/blob/master/session.vim